README.md

```diff
@@ -199,5 +202,6 @@ We have [guidelines for the directory structure of projects](https://github.com/
 - Simon Spies
 - Dominik Wehr
 - Maximilian Wuttke
+- Mark Koch
```

Comment:
we use alphabetic order here, so you can move yourself up :)

Reply:
Done ^^